.gitignore

```diff
@@ -21,3 +21,7 @@ module
 
 # Local Netlify folder
 .netlify
+
+package-lock.json
+yarn.lock
+
```

Comment:
```suggestion
```

I prefer adding lock files to git

Reply:
I've reverted `.gitignore` and re-committed `package-lock.json` and `yarn.lock` as you wanted